clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -3032,6 +3032,24 @@ private long getLatestLeaderOffsetAndHybridTopicOffset(
       latestLeaderOffset = 0;
     }
     long lag = lastOffsetInRealTimeTopic - latestLeaderOffset;
+
+    // Here we handle the case where the topic is actually empty,
```

Comment:
Should we call this part of the code only when `offsetLag > threshold`?

Reply:
I guess that'd make sense, but I'm not that worried about.  getPartitionOffsetLag should be based off of cached information anyhow.  So we're still only entering that if block if we've well and truly not consumed anything.